client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -193,8 +195,18 @@ class Dashboard extends Component {
 			isCalypsoStoreDeprecatedOrRemoved &&
 			shouldRedirectAfterInstall
 		) {
-			// Redirect to Core UI setup after finish installation.
-			this.redirectToWoocommerceSetup( selectedSite );
+			// Redirect to Core UI setup after finish installation if store is
+			// installed on this site. This check is needed because of an edge
+			// case where on a site where WooCommerce was installed then
+			// removed, then shouldRedirectAfterInstall is always true because
+			// the site option still includes WooCommerce - it isn't removed on
+			// plugin removal. See
+			// /client/extensions/woocommerce/state/sites/setup-choices/selectors.js
+			// in getFinishedInstallOfRequiredPlugins().
```

Comment:
The check makes sense, but the comment doesn't! What site option still includes WooCommerce?

Reply:
It made sense when I wrote it! Let me try again, I think I can simplify it.